test/hummingbot/strategy/test_script_strategy_base.py

```diff
@@ -0,0 +1,160 @@
+import unittest
+from decimal import Decimal
+from typing import List
+
+import pandas as pd
+
+from hummingbot.connector.exchange.paper_trade.paper_trade_exchange import QuantizationParams
+from hummingbot.core.clock import Clock
+from hummingbot.core.clock_mode import ClockMode
+from hummingbot.core.event.events import OrderType
+from hummingbot.strategy.market_trading_pair_tuple import MarketTradingPairTuple
+from hummingbot.strategy.script_strategy_base import ScriptStrategyBase
+from test.mock.mock_paper_exchange import MockPaperExchange
+
+
+class MockScriptStrategy(ScriptStrategyBase):
+    pass
+
+
+class ScriptStrategyBaseTest(unittest.TestCase):
+    level = 0
+    log_records = []
+    start: pd.Timestamp = pd.Timestamp("2019-01-01", tz="UTC")
+    end: pd.Timestamp = pd.Timestamp("2019-01-01 01:00:00", tz="UTC")
+    start_timestamp: float = start.timestamp()
+    end_timestamp: float = end.timestamp()
+    connector_name: str = "mock_paper_exchange"
+    trading_pair: str = "HBOT-USDT"
+    base_asset, quote_asset = trading_pair.split("-")
+    base_balance: int = 500
+    quote_balance: int = 5000
+    initial_mid_price: int = 100
+    clock_tick_size = 1
```

Comment:
We should put those in a class setup method for uniformity with the rest of our test modules

Reply:
Change applied